com.unity.render-pipelines.high-definition/Documentation~/HDRP-Sample-Content.md

```diff
@@ -40,5 +32,4 @@ This Sample includes various examples of lit and unlit particle effects.
 
 ![Material Samples](Images/MaterialSamples.png)
 
-This Sample includes various examples of Materials that use the [Lit Shader](Lit-Shader.html). The included Materials use effects such as subsurface scattering, displacement, and anisotropy. The **MaterialSamples** Scene requires Text Mesh Pro to display the text explanations.
-
+This Sample includes various examples of Materials. It includes materials that use the [Lit Shader](Lit-Shader.html), [Fabric Master Node](Master-Node-Fabric.html), [Hair Master Node](Master-Node-Hair.html) and [Decal Master Node](Master-Node-Decal.html). The included Materials use effects such as subsurface scattering, displacement, and anisotropy. The **MaterialSamples** Scene requires Text Mesh Pro to display the text explanations.
```

Comment:
Just a couple of things:
* In the second sentence "materials" -> "Materials".
* Oxford comma missing after [Hair Master Node](Master-Node-Hair.html). `[Hair Master Node](Master-Node-Hair.html), and [Decal Master Node](Master-Node-Decal.html).`

Reply:
good catch